src/main/scala/housekeeper/Dynamo.scala

```diff
@@ -1,41 +1,35 @@
 package housekeeper
 
-import com.amazonaws.regions.Regions
+import cats.implicits._
+import com.amazonaws.regions.Regions.EU_WEST_1
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClient
+import play.api.libs.json.Json
 
-object Dynamo {
+object Dynamo extends Logging {
 
-  val dyClient = AmazonDynamoDBAsyncClient.asyncBuilder().withRegion(Regions.EU_WEST_1).build()
+  val dyClient = AmazonDynamoDBAsyncClient.asyncBuilder().withRegion(EU_WEST_1).build()
 
   import com.gu.scanamo._
   import com.gu.scanamo.syntax._
 
-  case class Alert(
+  case class OphanAlert(
     email: String,
-    alertName: String,
-    query: Option[String],
-    threshold: Option[String],
-    filters: Option[String],
-    viewsPerMinute: Option[Int])
-
-  val table = Table[Alert]("ophan-alerts")
-
-  def deleteAlert(email: String): Unit = {
-
-    // we need to get all the alertNames for a certain email address:
-    val alertNamesForEmail = Scanamo.exec(dyClient)(table.query('email -> email))
-      .flatMap(_.toOption)
-      .map(_.alertName)
-
-    println("The following email alerts will be deleted:")
-    alertNamesForEmail foreach println
-    println("\n")
-
-    alertNamesForEmail foreach { aName =>
-      Scanamo.exec(dyClient)(table.delete('email -> email and 'alertName -> aName))
-    }
+    alertName: String
+  ) {
+    val primaryKey = 'email -> email and 'alertName -> alertName
+  }
 
-    println("Done! :)")
+  private val ophanAlertsTable = Table[OphanAlert]("ophan-alerts")
+
+  def deleteAllAlertsForEmailAddress(email: String): Unit = {
+    Scanamo.exec(dyClient)(for {
+      alertsForEmail <- ophanAlertsTable.query('email -> email)
+      _ = logger.info(Json.obj(
+        "alertsToDelete.emailAddress" -> email,
+        "alertsToDelete.found.count" -> alertsForEmail.size,
+      ))
+      _ <- alertsForEmail.flatMap(_.toOption).traverse(alert => ophanAlertsTable.delete(alert.primaryKey))
```

Comment:
nitpicking: `.collect { case Right(a) => a}` is a bit more descriptive than `.flatMap(_.toOption)`: one doesn't need to know what `Either.toOption` actually does